privacyidea/api/privacyideaserver.py

```diff
@@ -63,7 +64,7 @@ def create(identifier=None):
     param = request.all_data
     identifier = identifier.replace(" ", "_")
     url = getParam(param, "url", required)
-    tls = bool(int(getParam(param, "tls", default="1")))
```

Comment:
This replaces a custom conversions ``bool(int(<some string parameter>))`` with ``is_true``. Initially I thought this is a good idea :) But I'm not so sure if we should really do this, because it changes behavior: e.g. ``bool(int("2"))`` evaluates True, but ``is_true("2")`` is False. What do you think?